File: core/src/main/java/com/csc/core/base/BaseApplication.java
package com.csc.core.base;

import android.app.Application;
import android.content.Context;

import com.csc.core.manage.AccountManager;
import com.csc.core.util.ToastUtil;

public class BaseApplication extends Application {
    public static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = getApplicationContext();
        // Toast初始化
        ToastUtil.init(this);
        AccountManager.init(this);
    }
}

File: core/src/main/java/com/csc/core/base/BaseApplication.kt
package com.csc.core.base

import android.app.Application
import android.content.Context
import com.csc.core.manage.AccountManager
import com.csc.core.util.ToastUtil

open class BaseApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        mContext = applicationContext
        ToastUtil.init(this)
        AccountManager.init(this)
    }

    companion object {
        @JvmField
        var mContext: Context? = null
    }
}